README.md

```diff
@@ -88,6 +91,22 @@ will parse the raw cookie header and return an object `{ "sessionId": "aYb4uTIhd
 
 [cs]: https://www.npmjs.com/package/cookie#options-1
 
+<a name="rotating-secret"></a>
+### Rotating signing secret
+
+Key rotation is when you retire an encryption key and replace that old key by generating a new cryptographic key. To implement rotation, you can supply an `Array` of keys to `secret` option.
+
+**Example:**
+```js
+fastify.register(require('fastify-cookie'), {
+  secret: [key1, key2]
+})
+```
+
+Plugin will **always** use the first key to sign cookies. On the other hand, it will iterate over the supplied array to see if any of the available keys could decode the given signed cookie. This ensures that any old signed cookies are still valid.
```

Comment:
```suggestion
The plugin will **always** use the first key (`key1`) to sign cookies. When parsing incoming cookies, it will iterate over the supplied array to see if any of the available keys are able to decode the given signed cookie. This ensures that any old signed cookies are still valid.
```